CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+Add possibility of showing shipping cost above footer
+
+### Changed
+Rename `large` prop to `isSizeLarge` for better description
```

Comment:
I think `large` was ok, because, for boolean props, you could use:
`<Component large/>` that it would receive true, but ok 😄

Reply:
You can still do `<Component isSizeLarge />` 🤔

Reply:
But it aint clear on its purpose, is it a large font size, a large width, a large something else?

Reply:
It is now `isSizeLarge`. Better or worse?